README.md

```diff
@@ -22,7 +22,8 @@ Synology Package Repository
 8. Grant the created user with Package Administrator permissions `flask roles add admin@synocommunity.com package_admin`
 9. Grant the created user with Developer permissions `flask roles add admin@synocommunity.com developer`
 
-To clean data created by fake packages, run `flask spkrepo depopulate_db`
+To clean the database of fake packages, run `flask spkrepo depopulate_db`
```

Comment:
depopulate_db does both.

Reply:
Okay, cool

Reply:
Granted, the name isn't clear on that :shrug: 

Also, I'm very happy you got the tests to pass :+1:

Reply:
I'm glad too. Let's expedite the transition to production, aligning it with the committed changes in the repo, and then proceed to enhance with new features.

Reply:
I intend to download the db from production and run some tests locally just to make sure. I'll have to see if I can push the new docker images or if @Diaoul has to it.